### ISSUE TITLE
Bomb Box ESP

### ISSUE THREAD

**Issue by Dimitriblyater** [XBoom31/ApocalyseCheatsSC#31]
Bomb box esp seems to get broken after recent update

http://i.imgur.com/nhrD9ZT.png

**Comment by XBoom31** [comment]
uhm, that was fixed two days ago, update the cheat and tell me if you still got this prob

**Comment by Dimitriblyater** [comment]
What exactly was causing this issue?

The cheat is up to date from this https://github.com/XBoom31/ApocalyseCheatsSC/commit/bbec25d666acc37c056ec640c8aa995752d20b11

and the recent commits I added manually

**Comment by XBoom31** [comment]
this can cause the issue
`VertLine = (((float)size.w) * (0.2f));`
But I played with this update 1 h ago and all was fine

**Comment by XBoom31** [comment]
https://pastebin.com/Xu67xQSX
like this https://gyazo.com/57a04f1bd517777a8906efaab36ee1eb

**Comment by Dimitriblyater** [comment]
I copied over the whole ESP.cpp
Same problem, idk why.

**Comment by Dimitriblyater** [comment]
I copied over the whole ESP.cpp
I still have the issue

I did a temporary fix by disabling the esp box and only keeping the bomb text.
I don't know what is causing this because I have the same code as your esp.cpp

**Comment by XBoom31** [comment]
![730_screenshots_20170626133748_1](https://user-images.githubusercontent.com/27855611/27535786-d56d9d94-5a74-11e7-8550-dd6f8e0df7e8.jpg)
for me works, idk

**Comment by Dimitriblyater** [comment]
I found the problem.
`void DEsp::Move(CUserCmd *pCmd, bool &bSendPacket)`

The space next to *pCmd and bool had to be removed. Aswell as some other spaces in the code.
I don't know why there were spaces there since I pasted in the whole esp.cpp
I think something just auto adds the spaces because I never changed anything.

Thank you for trying to help me solve this issue and thanks for updating!
:D
